packages/mux-audio/test/player.test.js

```diff
@@ -19,4 +19,42 @@ describe('<mux-audio>', () => {
     assert.equal(player.preferMse, true, 'prefer-mse is on');
     assert.equal(player.debug, false, 'debug is off');
   });
+
+  it('dispatches events properly', async function () {
+    const player = await fixture(`<mux-audio
+      playback-id="vDpm5ygrRJgfIEPNIc02IJR4Trf3z00AiP"
+      muted
+    ></mux-audio>`);
+
+    const eventMap = {};
+    AudioEvents.forEach((type) => {
+      eventMap[type] = false;
+      player.addEventListener(type, (e) => {
+        assert.equal(e.target, player);
+        eventMap[e.type] = true;
+      });
+    });
+
+    player.volume = 0.5;
+
+    // Seems only <audio> is throwing in my tests:
+    //
+    //   NotAllowedError: play() failed because the user didn't interact
```

Comment:
I'm surprised this was an issue, even though it was muted. Maybe an assumption that `<audio>` element shouldn't be muted?

Reply:
yes, could be. I was surprised too